internal/feather.go

```diff
@@ -524,30 +530,36 @@ func ChalkMarshal(attrs map[string]any) ([]byte, error) {
 	// Magic string header
 	_, err := ioWriter.WriteString("CHALK_BYTE_TRANSMISSION")
 	if err != nil {
-		return nil, err
+		return nil, errors.Wrap(err, "failed to write chalk transmission magic string")
```

Comment:
nit: I see both fmt.Errorf and errors.Wrap

Reply:
thank you